File: src/main/java/io/github/support/terminal/application/config/SpaRoutingController.java
package io.github.support.terminal.application.config;


import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
public class SpaRoutingController {
    //This is for embedded UI
    @RequestMapping(value = "/{[path:[^\\.]*}")
    public String redirect() {
        return "forward:/";
    }
}

File: src/main/java/io/github/bot/terminal/application/config/SpaRoutingController.kt
package io.github.bot.terminal.application.config

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping

@Controller
class SpaRoutingController {
    //This is for embedded UI
    @RequestMapping(value = ["/{[path:[^\\.]*}"])
    fun redirect(): String {
        return "forward:/"
    }
}